src/login/logind-dbus.c

```diff
@@ -1512,6 +1512,9 @@ static int bus_manager_log_shutdown(
         } else if (streq(unit_name, SPECIAL_KEXEC_TARGET)) {
                 p = "MESSAGE=System is rebooting with kexec";
                 q = "SHUTDOWN=kexec";
+        } else if (streq(unit_name, SPECIAL_FACTORY_RESET_TARGET)) {
+                p = "MESSAGE=System is performing factory reset";
+                q = "SHUTDOWN=factory-reset";
```

Comment:
this is not a form of shutdown, i don't think we should claim so. please leave `q` as NULL here. nd use a different MESSAGE_ID (register a new one maybe)